src/main/scala/tech/cryptonomic/conseil/tezos/BlockOperations.scala

```diff
@@ -0,0 +1,35 @@
+package tech.cryptonomic.conseil.tezos
+import TezosTypes.{
+  Block,
+  Delegation,
+  InternalOperationResults,
+  Operation,
+  OperationsGroup,
+  Origination,
+  Reveal,
+  Transaction
+}
+
+object BlockOperations {
```

Comment:
Please leave a ScalaDoc comment for this object.

Reply:
Removed the class altogether